src/withCustomComponent.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { Text } from 'react-native';
+import PropTypes from 'prop-types';
+
+const withCustomComponent = (Comp) =>
+  class extends Component {
+    static propTypes = {
+      component: PropTypes.node,
+    }
+
+    setNativeProps = nativeProps => {
+      /* eslint no-underscore-dangle: 0 */
+
+      this._root.setNativeProps(nativeProps);
```

Comment:
This will not fix bug https://github.com/frostney/react-intl-native/issues/12
Remove this method.
Please add me as collaborator and I will fix that issue in separate pull request.

Reply:
Sure thing, I probably misunderstand you on this.